packages/compartment-mapper/test/scaffold.js

```diff
@@ -17,6 +17,12 @@ import { makeReadPowers } from '../src/node-powers.js';
 
 export const readPowers = makeReadPowers({ fs, crypto, url });
 
+export const sanitizePaths = (text = '', tolerateLineChange = false) => {
+  if (tolerateLineChange) {
+    text = text.replace(/:[0-9]+:[0-9]+/g, ':…');
+  }
+  return text.replace(/file:\/\/[^'"\n]+\/endo\/packages\//g, 'file://.../');
```

Comment:
Please don't assume the repository has been checked out in a directory called `endo`, but since the `packages` directory is within the repo, I think it's fine to use it as the indicator of a path.

```suggestion
  return text.replace(/file:\/\/[^'"\n]+\/packages\//g, 'file://.../');
```